### PR TITLE
Backend daily schedule

### PR DESCRIPTION
**Pull request by Siramul**
Models, views, serializers and URLs were added/modified for daily schedules. POST, PUT, GET & DELETE requests were completed for daily schedule.

### CODE REVIEW

File: express_cut/express_service/views.py [hunk 1]
```diff
@@ -91,5 +91,58 @@ def users_views(request, pk):
...
+
+@swagger_auto_schema(methods=['GET', 'DELETE'], responses={**swagResp.commonResponses, **swagResp.getResponse(UserSerializer)},tags=['dailySchedule'],)
+@swagger_auto_schema(methods=['PUT'], request_body=DailyScheduleSerializer, responses=swagResp.commonPOSTResponses,
+                     tags=['dailySchedule'], )
+@api_view(['GET','PUT', 'DELETE'])
+@authentication_classes([SessionAuthentication, BasicAuthentication])
+@permission_classes([IsAuthenticated])
+def schedule_views_put(request, pk):
+    try:
+        schedule = DailySchedule.objects.get(pk=pk)
```
Comment (yarm369) [inline express_cut/express_service/views.py:124]: I don't see that your are managing permissions here. Everyone shouldn't have permissions for this. They should look something like this:

GET - Managers, Customers and the stylist that is related to that schedule.
PUT- Managers
DELETE - Managers

File: express_cut/express_service/views.py [hunk 2]
```diff
@@ -91,5 +91,58 @@ def users_views(request, pk):
...
+    if request.method == 'GET':
+        serializer = DailyScheduleSerializer(schedule)
+        return Response(serializer.data)
+
+    elif request.method == 'PUT':
+        data = request.data
+        serializer = DailyScheduleSerializer(schedule, data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data)
```
Comment (yarm369) [inline express_cut/express_service/views.py]: Missing HTTP Status code

Comment (yarm369) [reply express_cut/express_service/views.py]: I see it's also missing on other previous implemented function (not related to this PR), please add them.

File: express_cut/express_service/serializers.py [hunk 3]
```diff
@@ -19,3 +20,10 @@ def create(self, validated_data):
         validated_data['password'] = make_password(validated_data.get('password'))
         return User.objects.create(**validated_data)
 
+
+class DailyScheduleSerializer(serializers.ModelSerializer):
```
Comment (yarm369) [inline express_cut/express_service/serializers.py:24]: Consider Handling here the timeslots. Lets say I want to PUT a schedule, shouldn't I be able to send in the request the related timeslots?  Or If I wanted to do a GET, shouldn't I receive in the response the timeslots related to that schedule. Something like this:
`
{
    "date": "2020-10-25",
    "stylist_id": 0,
    "time_slot": {
        {
            "start_time": "07:00:00",
            "end_time": "12:00:00"
        },
        {
            "start_time": "13:00:00",
            "end_time": "17:00:00"
        },
    }
}`

**Review by yarm369 — CHANGES_REQUESTED** [review]
Left a few comments you should check before reviewing. In general seems good, but missing Permissions for GET, PUT, DELETE and should consider that a Schedule is composed of **multiple** timeslots.

File: express_cut/express_service/views.py (hunk 1 above)
Comment (Siramul) [reply express_cut/express_service/views.py:124]: Everyone should be able to see every Stylist schedule. Will handle permissions for PUT and DELETE.

File: express_cut/express_service/views.py (hunk 2 above)
Comment (Siramul) [reply express_cut/express_service/views.py]: OK, will add.

File: express_cut/express_service/serializers.py (hunk 3 above)
Comment (yarm369) [reply express_cut/express_service/serializers.py:24]: This is pending. Will work on it in a new issue. (Reference: https://www.django-rest-framework.org/api-guide/serializers/#dealing-with-nested-objects)

**Review by yarm369 — APPROVED** [review]
Looks Good!

**Pull request merged**